plugins/woocommerce-admin/client/marketplace/components/category-selector/category-selector.tsx

```diff
@@ -104,10 +104,13 @@ export default function CategorySelector(): JSX.Element {
 
 	if ( isLoading ) {
 		return (
-			<div className="woocommerce-marketplace__category-selector-loading">
-				<p>{ __( 'Loading categories…', 'woocommerce' ) }</p>
-				<Spinner />
-			</div>
+			<ul className="woocommerce-marketplace__category-selector">
+				{[...Array(5)].map(() =>
+					<li className="woocommerce-marketplace__category-item">
+						<CategoryLink slug="" label="" selected={ false } />
+					</li>
+				)}
```

Comment:
Nice! The attention to detail on even showing a skeleton category selector is ❤️ 

There are some linting issues. Suggestion below.

```suggestion
				{ [ ...Array( 5 ) ].map( () => (
					<li className="woocommerce-marketplace__category-item">
						<CategoryLink slug="" label="" selected={ false } />
					</li>
				) ) }
```

Also, I get a missing key JSX error for the list items. Not sure it's strictly needed in this particular case.

Reply:
Yeah, I flip-flopped on adding a key. Obviously it's not needed because we don't mind if the entire container needs re-rendering on a change (the only change that can happen requires a re-render of the container _anyway_!), but if the warning bugs people we could fix it...